packages/circuits/circuits/qr_verify.circom

```diff
@@ -5,11 +5,14 @@ include "./sha.circom";
 
 
 template QR_Verify(n, k, len) {
+    signal input signedMessage;
     signal input padded_message[len]; // private
     signal input message_len; // private 
     signal input signature[k]; //private
     signal input modulus[k]; //public
 
+    signal output messageSquare;
```

Comment:
Does it need to be an output signal? The proof would change based on the input either way right?

Reply:
Yeah, you right! We don't need output signal.